model-optimizer/extensions/ops/BatchNormTraining.py

```diff
@@ -0,0 +1,42 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from mo.graph.graph import Graph
+from mo.ops.op import Op
+import numpy as np
+
+
+class BatchNormTraining(Op):
+    """
+    BatchNormInference will be replaced by BatchNormInference after 
+        FusedBatchNormTraining tranformation 
+    """
+    op = 'batchNormTraining'
```

Comment:
Let's change the first character to capital 'B'. This is our standard approach. The "batchNormInference" is some strange exception.

Reply:
Change to capital `B`.

Reply:
done